### PR TITLE
SBERDOMA-1027 Add employee specializations input

### PR DESCRIPTION
**Pull request by MrFoxPro**

### CODE REVIEW

File: apps/condo/domains/organization/components/CreateEmployeeForm.tsx [hunk 1]
```diff
@@ -96,6 +103,18 @@ export const CreateEmployeeForm: React.FC = () => {
...
+    const selectedRole = useMemo(() => employeeRoles.find(x=> x.id === selectedRoleId), [selectedRoleId])
+
+    const searchClassifers = (_, input) => 
+        classifiersLoader.search(input, TicketClassifierTypes.category)
+            .then(result=>result.map((classifier)=> ({ text: classifier.name, value: classifier.id })))
+
+    useEffect(()=> {
+        classifiersLoader.init()
+        return () => classifiersLoader.clear()
+    })
```
[the remark this answers is not in the document]
Comment (AntonAL) [reply apps/condo/domains/organization/components/CreateEmployeeForm.tsx]: Added in `a7bf497`

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js [hunk 2]
```diff
@@ -18,11 +18,11 @@ const InviteNewOrganizationEmployeeService = new GQLCustomSchema('InviteNewOrgan
     types: [
         {
             access: true,
-            type: 'input InviteNewOrganizationEmployeeInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, email: String!, phone: String, name: String, role: OrganizationEmployeeWhereUniqueInput, position: String}',
+            type: 'input InviteNewOrganizationEmployeeInput { dv: Int!, sender: JSON!, organization: OrganizationWhereUniqueInput!, email: String!, phone: String, name: String, role: OrganizationEmployeeWhereUniqueInput, position: String, specializations: [TicketCategoryClassifierWhereInput!]}',
         },
         {
             access: true,
-            type: 'input ReInviteOrganizationEmployeeInput { dv: Int!, sender: SenderFieldInput!, organization: OrganizationWhereUniqueInput!, email: String!, phone: String}',
+            type: 'input ReInviteOrganizationEmployeeInput { dv: Int!, sender: JSON!, organization: OrganizationWhereUniqueInput!, email: String!, phone: String}',
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js:25]: REBASE IT!

**Review by pahaz — CHANGES_REQUESTED** [review]

File: apps/condo/domains/organization/components/CreateEmployeeForm.tsx [hunk 3]
```diff
@@ -96,6 +100,18 @@ export const CreateEmployeeForm: React.FC = () => {
         role: get(employeeRoles, [0, 'id'], ''),
     }
 
+    const [selectedRoleId, setSelectedRoleId] = useState(initialValues.role)
+    const selectedRole = useMemo(() => employeeRoles.find(x=> x.id === selectedRoleId), [selectedRoleId])
```
Comment (Dimitreee) [inline apps/condo/domains/organization/components/CreateEmployeeForm.tsx:104]: add readable virable names

File: apps/condo/domains/user/components/NotDefinedField.tsx [hunk 4]
```diff
@@ -2,10 +2,12 @@ import { Typography } from 'antd'
 import React, { ReactElement } from 'react'
 import { useIntl } from '@core/next/intl'
 
+type Value = string | any[]
```
Comment (Dimitreee) [inline apps/condo/domains/user/components/NotDefinedField.tsx:5]: add readable varibale names

File: apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js (hunk 2 above)
Comment (AntonAL) [reply apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js]: Rebased

File: apps/condo/domains/user/components/NotDefinedField.tsx (hunk 4 above)
Comment (AntonAL) [reply apps/condo/domains/user/components/NotDefinedField.tsx]: Renamed in `fed18998b`

File: apps/condo/domains/organization/components/CreateEmployeeForm.tsx (hunk 3 above)
Comment (Dimitreee) [reply apps/condo/domains/organization/components/CreateEmployeeForm.tsx:104]: @AntonAL ```find(x=> x.id === selectedRoleId)```

**Review by Dimitreee — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/organization/components/CreateEmployeeForm.tsx (hunk 3 above)
Comment (AntonAL) [reply apps/condo/domains/organization/components/CreateEmployeeForm.tsx:104]: @MrFoxPro, please, use meaningful variable names.

Comment (AntonAL) [reply apps/condo/domains/organization/components/CreateEmployeeForm.tsx]: Refactored in `dfba0dd80`

**Pull request merged**